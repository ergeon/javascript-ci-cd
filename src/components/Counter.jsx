import React, { Component } from "react";

class Counter extends Component {

  renderSomething() {
    return <div>Something</div>;
  }

  render() {
    return (
      <div>
        Counter
      </div>
    );
  }
}

export default Counter;
