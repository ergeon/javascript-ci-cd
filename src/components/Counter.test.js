import React from 'react';
import renderer from 'react-test-renderer';
import Counter from './Counter';


test('Counter dummy test', () => {
  const component = renderer.create(
    <Counter />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
