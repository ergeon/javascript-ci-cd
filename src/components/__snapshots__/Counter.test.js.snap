// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter dummy test 1`] = `
<div>
  Counter
</div>
`;
